sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_connection.py

```diff
@@ -135,7 +135,7 @@ def __init__( # pylint:disable=too-many-locals,too-many-statements
             custom_endpoint = f"{custom_parsed_url.hostname}:{custom_port}{custom_parsed_url.path}"
         self._container_id = container_id or str(uuid.uuid4())
         self._network_trace = network_trace
-        self._network_trace_params = {"amqpConnection": self._container_id, "amqpSession": None, "amqpLink": None}
+        self._network_trace_params = {"amqpConnection": self._container_id}
```

Comment:
didn't we add this to fix another bug in our set up ? Would this cause it to resurface those errors?

Reply:
I don't think so?